src/main/resources/db/migration/V27__refine_hyundai_z_work_merchant_targets.sql

```
@@ -0,0 +1,43 @@
+JOIN card_content_versions version ON version.content_version_id = benefit.content_version_id
+JOIN cards card ON card.card_id = version.card_id
+WHERE card.gorilla_card_id = '2680'
+  AND offer.offer_name IN ('편의점 10% 청구 할인', '커피전문점 10% 청구 할인')
+  AND target.target_type = 'merchant_category';
+
+INSERT INTO benefit_rule_targets
+    (target_id, rule_id, condition_group, match_mode, target_type,
+     merchant_category_id, merchant_id, target_code, target_name,
+     target_source, target_authority, minimum_place_confidence, created_at, updated_at)
+SELECT UUID(), rule_data.rule_id, brands.condition_group, 'include', 'merchant',
+       NULL, merchant.merchant_id, brands.merchant_name, brands.merchant_name,
```

SQLFluff (4.2.2)

[error] Reuse of column alias 'merchant_name' from line 18.

(AL08)